src/main.rs

```diff
@@ -261,6 +262,37 @@ fn output_json<W: io::Write>(
     Ok(())
 }
 
+
+fn output_openbgpd<W: io::Write>(
+    roas: &AddressOrigins,
+    output: &mut W
+) -> Result<(), ProcessingError> {
+    let mut first = true;
+    writeln!(output, "roa-set {{")?;
+    for addr in roas.iter() {
+        if first {
+            first = false
+        }
+        else {
+            write!(output, "\n")?;
+        }
+        write!(output, "    {}/{}",
+            addr.address(), addr.address_length(),
+        )?;
+        if addr.address_length() < addr.max_length() {
+            write!(output, " maxlen {}",
+                addr.max_length(),
+            )?;
+        }
+        write!(output, " source-as {}",
+            &addr.as_id().to_string()[2..], // sorry, this feels ugly
```

Comment:
I _think_ this should work instead without double string conversion:

```suggestion
            u32::from(addr.as_id())
```